app/uk/gov/hmrc/fhregistrationfrontend/views/declaration.scala.html

```diff
@@ -48,158 +47,166 @@
+      errorMessage = if(declarationForm.error(s"$alternativeEmailKey.email") || declarationForm.error(s"$alternativeEmailKey.emailConfirmation")) {
+        declarationForm.error(s"$alternativeEmailKey.email").map{ e =>
+          ErrorMessage(
+            content = Text(messages(s"fh.declaration.${e.key}.${e.message}"))
+          )
 
-  @error_box(declarationForm,".declaration.")
+        }
+      } else None,
+      value = declarationForm(s"$alternativeEmailKey.email").value,
+    ))
+
+    @viewHelpers.govukInput(Input(
+      id = s"$alternativeEmailKey.emailConfirmation",
+      name = s"$alternativeEmailKey.emailConfirmation",
+      inputType = "email",
+      label = Label(
+        isPageHeading = true,
+        content = Text(messages("fh.confirmationEmail.email.confirmation.label"))
+      ),
+      autocomplete = Some("email"),
+      spellcheck = Some(false),
+      classes = "govuk-!-width-one-half",
+      errorMessage =
+        declarationForm.error(s"$alternativeEmailKey.emailConfirmation").map{ e =>
+          ErrorMessage(
+            content  = Text(messages(s"fh.declaration.${e.key}.${e.message}"))
+          )
 
-  @helper.form(action = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.DeclarationController.submitForm, 'novalidate -> "novalidate") {
+        },
+      value = declarationForm(s"$alternativeEmailKey.emailConfirmation").value,
+    ))
+  </div>
+}
 
-    @helper.CSRF.formField
+@layout(title = titlePrefix + title) {
 
-    @showAlert
+  @viewHelpers.form(action = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.DeclarationController.submitForm) {
+    @if(declarationForm.errors.nonEmpty) {
 
+      @viewHelpers.govukErrorSummary(ErrorSummary(errorList =
+        declarationForm.errors.map { error =>
+          ErrorLink(
+            href = Some(s"#${error.key}"),
+            content = Text(messages(s"fh.declaration.${error.key}.${error.message}"))
+          )
+        }
+        ,
+        title = Text(messages("generic.errorSummary"))))
+    }
     @PageHeading(
       PageHeadingParams(
         heading = title
       )
     )
-
-    @TextInput(
-      TextInputParams(
-        name = fullNameKey,
-        label = Messages("fh.declaration.person_name.label"),
-        value = declarationForm(fullNameKey).value,
-        error = Messages("fh.declaration.person_name." + declarationForm(fullNameKey).error.fold("")(_.message)),
-        hasErrors = declarationForm(fullNameKey).hasErrors,
-        autocomplete = Some("name"),
-        spellcheck = false
-      )
-    )
-
-    @TextInput(
-      TextInputParams(
-        name = jobTitleKey,
-        label = Messages("fh.declaration.person_status.label"),
-        value = declarationForm(jobTitleKey).value,
-        error = Messages("fh.declaration.person_status." + declarationForm(jobTitleKey).error.fold("")(_.message)),
-        hasErrors = declarationForm(jobTitleKey).hasErrors
-      )
-    )
-
-    <h2 class="small-heading">@{
+    @showAlert
```

Comment:
Not sure what difference it makes without going running the code but the showAlert is now after the page heading rather than before it.

Reply:
Yes because as per new component we show Alert after the Page heading and not above them. The design team has confirmed that

Reply:
ok